about/index.html

```diff
@@ -3,24 +3,21 @@
 title: About Us
 meta_description: OME produces open tools to support data management for microscopy. Learn more here.
 funding:
-    - grant_name: "BBSRC BBR Award"
```

Comment:
Is this "previous funding"?

Reply:
Definitely previous funding. Would you keep a no-op dictionary for historical funding?

As an aside, I looked into https://schema.org/Grant which would be an interesting way forward and would likely capture the main concepts. Two caveats: this element is not integrated in the official schema.org spec yet and I have not found a good way to capture active/previous concepts.

Reply:
> Would you keep a no-op dictionary for historical funding?

Yes. Seems a shame to lose it.

Reply:
95d469f